src/aspire/source/__init__.py

```diff
@@ -14,7 +16,7 @@
     Pipeline,
 )
 from aspire.operators import LambdaFilter, MultiplicativeFilter, PowerFilter
-from aspire.storage import save_star
```

Comment:
So this function is no longer used? We should probably remove it in this PR then since the two new `ImageSource` methods replace it.

Reply:
I checked it. It is ok to remove. Done in commit 2cae866.